R/format_data.R

```diff
@@ -135,6 +153,52 @@ fmt_number <- function(data,
     sep_mark <- ""
   }
 
+  # Determine whether large-number suffixing is being
+  # used and set the appropriate inputs
+  if (is.logical(suffixing) &&
+      length(suffixing) == 1 &&
+      suffixing) {
+
+    # If `suffixing` is TRUE, the `to_suffix`
+    # logical will also be set to TRUE and the
+    # default set of symbols will be assigned
+    # to `num_suffixes`
+    to_suffix <- TRUE
+    num_suffixes <- c("K", "M", "B", "T")
+
+  } else if (is.logical(suffixing) &&
+             length(suffixing) == 1 &&
+             suffixing == FALSE) {
+
+    # If `suffixing` is FALSE, the `to_suffix`
+    # logical will also be set to FALSE
+    to_suffix <- FALSE
+
+  } else if (is.character(suffixing) &&
+             length(suffixing) == 4){
```

Comment:
Check that vector is unnamed (so we can use names later to mean other things)

Reply:
I used the following to `stop()` even if there are partially-named vectors:

```r
} else if (is.character(suffixing) &&
             length(suffixing) == 4){

    # Stop function if the character vector `suffixing`
    # contains any names
    if (!is.null(names(suffixing))) {
      stop("The character vector supplied to `suffixed` cannot contain names.",
           call. = FALSE)
    }

...
}
```